doc/source/2-HOW-TO/CASE1-REVERSE-ENGINEERING-PACKET-TRACER-FOR-CRYPTANALYSIS-ATTACK-EXPLOIT-DEV.rst

```diff
@@ -0,0 +1,9 @@
+Case 1: Reverse Engineering Packet tracer in order to code exploits to break cryptographic algorithm: Reverse Engineering, analysis and cryptanalysis.
```

Comment:
I supposed this is based on the binary you were trying to analyze. I don't think we should add anything until we get it working.